docs/README-develop-beaker-scripts.md

```diff
@@ -961,6 +961,7 @@ Custom keys may be added for various unique test requirements but they should be
 |:-----------------|:---------------
 | `:asn`           | An Autonomous-System number. Used in some routing protocol tests.
 | `:encap_prof_global` | The global encapsulation profile configuration. This configuration is a dependency for testing with `cisco_interface_service_vni`.
+| `:ensure`        | Allowed values: `:unsupported`. This key is only used for providers that do not support ensurability; this prevents the test_harness from inserting an ensure value into the manifest. Also see Test Case key `:ensure`.
```

Comment:
This seems a bit confusing to me (`:ensure` means one thing at top level, a different thing at test case level). Would it be clearer and more meaningful if this new top-level key is called `:ensurable`?